test/addons/parse-encoding/test.js

```diff
@@ -8,6 +8,7 @@ assert.strictEqual(parseEncoding(''), 'UNKNOWN');
 
 assert.strictEqual(parseEncoding('ascii'), 'ASCII');
 assert.strictEqual(parseEncoding('base64'), 'BASE64');
+assert.strictEqual(parseEncoding('base64url'), 'BASE64URL');
```

Comment:
I think you also need to add this to `ENCODING_MAP` in the `.cc` file here :)

Reply:
I did, locally. But i need to rebuild them somehow :)

Reply:
`../../../deps/npm/node_modules/node-gyp/bin/node-gyp.js --nodedir=../../.. rebuild`, from the directory of the test (yes, I have that saved as an alias :yum:)

Reply:
See, i did that and it still fails. In CI too :(

Reply:
possibly src/api/encoding.cc?

Reply:
@panva Was just about to say that :) Yes, that parser also needs to be updated here, it seems

Reply:
I'm certain about the second part of [the diff](https://github.com/nodejs/node/compare/047efc92cfae77157c67991532356656e4eb84ce..dcbd9264c14711e2705a47bd2cb38f0e5bbc6880), not so much about the first one - do you see a better way to optimize the distinction between `se64` and `se64url`?